src/cpp/src/visual_language/pipeline.cpp

```diff
@@ -324,19 +324,39 @@ class VLMPipeline::VLMPipelineImpl : public VLMPipelineBase{
 VLMPipeline::VLMPipeline(
     const std::filesystem::path& models_dir,
     const std::string& device,
-    const ov::AnyMap& properties
+    const ov::AnyMap& user_properties
 ) {
     auto start_time = std::chrono::steady_clock::now();
+    auto [properties, attention_backend] = utils::extract_attention_backend(user_properties);
 
-    if (properties.find(scheduler_config.name()) != properties.end() ||
-        properties.find(utils::DRAFT_MODEL_ARG_NAME) != properties.end() ||
-        properties.find(prompt_lookup.name()) != properties.end()) {
-        auto [plugin_config, scheduler_config] = utils::extract_scheduler_config(properties);
-        m_pimpl = std::make_unique<VLMContinuousBatchingAdapter>(models_dir, scheduler_config, device, plugin_config);
+    // If CB is invoked explicitly, create CB adapter as is and re-throw in case if internal issues
+    if (utils::explicitly_requires_paged_attention(properties)) {
+        auto [plugin_properties, scheduler_config] = utils::extract_scheduler_config(properties, utils::get_latency_oriented_scheduler_config());
+        m_pimpl = std::make_unique<VLMContinuousBatchingAdapter>(models_dir, scheduler_config, device, plugin_properties);
     }
-    else {
+
+    if (m_pimpl == nullptr && device == "NPU") {
+        m_pimpl = std::make_unique<VLMPipelineImpl>(models_dir, device, properties);
+    }
+
+    // try to call CB adapter one more time, but with safe guard to silent exception
+    if (m_pimpl == nullptr && attention_backend == PA_BACKEND) {
+        try {
+            auto [plugin_properties, scheduler_config] = utils::extract_scheduler_config(properties, utils::get_latency_oriented_scheduler_config());
+            // we need use CB only for x86 and arm64, as for other architectures like risc-v we can create Paged Attention based model
+            // but cannot perform its inference later
+#if defined(OPENVINO_ARCH_X86_64) || defined(OPENVINO_ARCH_ARM64)
+            m_pimpl = std::make_unique<VLMContinuousBatchingAdapter>(models_dir, scheduler_config, device, plugin_properties);
+#endif
+        } catch (ov::Exception&) {
```

Comment:
We'd probably need perf and accuracy validation runs. Please ensure llm_bench and wwb are able to enforce CB with using ATTENTION_BACKEND via config if they aren't. (They are probably fine)

Reply:
Yes, both llm_bench and wwb are able to enforce CB with using ATTENTION_BACKEND via config. 

Andrei made validation runs for performance and accuracy. According his report, there's no degradations in accuracy, regarding performance there's run-to-run deviations of Qwen2VL performance +-20%, 
the rest works approximately as stateful or have performance improvements comparing to stateful.